cloud-common/src/main/java/com/sequenceiq/cloudbreak/cloud/PlatformParametersConsts.java

```diff
@@ -2,6 +2,7 @@
 
 public class PlatformParametersConsts {
 
+    // TTL unit: milliseconds
```

Comment:
Would be change the name to `TTL_IN_MILLIS` for example to avoid comment

Reply:
done